src/integration/pages/articlePage/mundo.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @pathname /mundo/articles/ce42wzqr2mko
```

Comment:
Ingenious! I like it 😁

Reply:
that's called a docblock pragma in case you were wondering!